test/test06/Makefile

```diff
@@ -0,0 +1,4 @@
+GOTARGETS=
```

Comment:
Just to make sure I understand the tests, these cases are just intended to make sure the commands succeed, but they don't assert any expectations?

Reply:
There are some assertions in

https://github.com/AchievementNetwork/common-make/blob/main/test/test.sh

but they don't cover this test case because building it creates no executable to check for the presence of